Plugins/Flow.Launcher.Plugin.QuickLook/Flow.Launcher.Plugin.QuickLook.csproj

```diff
@@ -0,0 +1,51 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
If we merge is this plugin going to be a default one?

Reply:
No I will take it out before merge. There is already a QL plugin repo in the org.